MantidPlot/make_package.rb.in

```diff
@@ -339,7 +339,7 @@ system_python_extras = "/System/Library/Frameworks/Python.framework/Versions/2.7
 pip_site_packages = "/Library/Python/2.7/site-packages"
 directories = ["sphinx","sphinx_bootstrap_theme","IPython","zmq","pygments","backports", "qtawesome", "qtpy",
                "certifi","tornado","markupsafe","matplotlib","mpl_toolkits", "jinja2","jsonschema","functools32",
-               "ptyprocess","CifFile","yaml","requests"]
+               "ptyprocess","CifFile","yaml","requests","skimage"]
```

Comment:
Will this correctly pull in all of the dependencies?

Reply:
Looks like packaging will require more work...

Reply:
Added everything suggested by `homebrew-pypi-poet`.